catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/FeedbackApplication.java

```diff
@@ -81,6 +81,8 @@ public void init() {
             String emailBody = getEmailBody(feedback);
             if (emailBody != null) {
               emailBody = emailBody.replaceAll("\\\\n", "\n");
+            } else {
+              emailBody = "";
```

Comment:
❓ On line 92 we set the message content to `emailBody` and set the content type to `text/html`. I don't think a blank string is valid text/html. Perhaps we should choose something other than blank string for the default. 

This also begs the question - Do we want to send an email at all if there's no body content?

Reply:
`<html/>`